Tests/Shared/SampleData/CustomAttributesSampleData.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+public enum CustomAttributesSampleData {
+    // Common set of custom attributes that tries to demonstrate all of the different ways the customers might use
+    // attributes to verify we support it all.
+    public static let givenCustomAttributes: [String: Any] = [
+        "firstName": "Dana",
+        "last_name": "Green",
+        "HOBBY": "football",
+        "nested": [
+            "is adult": true,
```

Comment:
Is the space in `is adult` intentional ?

Reply:
Yes. I wanted to test this scenario in case a customer wanted to send us an attribute with a space in it.